pkg/scheduler/cache/cache.go

```diff
@@ -851,4 +858,17 @@ func (sc *SchedulerCache) recordPodGroupEvent(podGroup *schedulingapi.PodGroup,
 		return
 	}
 	sc.Recorder.Eventf(pg, eventType, reason, msg)
+
+	jobLister := sc.jobInformer.Lister()
+	batchJob, err := jobLister.Jobs(podGroup.Namespace).Get(podGroup.Name)
```

Comment:
we should not bind podgroup to job

Reply:
So if the podgroup has an error, should the error information be synchronized to the job?

Reply:
User current needs to describe resources one by one

Reply:
The job controller/operator should monitor PodGroup's event.

Reply:
So , i should change to monitor podgroup event in controller ,right?

Reply:
Yes, The podgroup is created by controller. The controller monitor podgroup event seems reasonable. 
Job should not be involved in scheduler because we need to keep compatible with variable operator without using volcano job.

Reply:
I have move code to job controller. PLTA